src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -120,3 +120,46 @@ describe('generateConfluenceMarkdown function', () => {
         expect(results).not.toContain(`|[${singleIssue[0].title}|${singleIssue[0].url}]|`);
     });
 });
+
+describe('cleanMarkdown function', () => {
+    it('should return string without altering it', () => {
+        const title1 =
+            '|[2020 upgrade edition menus too narrow|https://github.com/marmelab/react-admin/issues/7520]|8&nbsp;days|';
+        const cleanedMarkdown = cleanMarkdown(title1);
+        expect(cleanedMarkdown).toMatch(
+            '|[2020 upgrade edition menus too narrow|https://github.com/marmelab/react-admin/issues/7520]|8&nbsp;days|'
```

Comment:
This could expect toMatch(title1). That would make it clearer that no changes are expected.

Reply:
Why is this toMatch and not toEqual?